src/custom/components/SideBanner/index.tsx

```diff
@@ -129,23 +134,24 @@ const FooterContent = styled.div`
   }
 `
 
-export default function SideBanner({ type }: BannerProps) {
-  const [isActive, setIsActive] = useState(true)
+export default function SideBanner({ type, isVisible }: BannerProps) {
+  const [isActive, setIsActive] = useState(isVisible)
 
   // const isHidden = !isVisible || !isActive
 
   const handleClose = () => {
     setIsActive(false)
   }
 
-  // useEffect(() => {
-  //   if (isNotificationClosed !== null) {
-  //     setIsActive(!isNotificationClosed)
-  //   }
-  // }, [isNotificationClosed, isActive])
+  useEffect(() => {
+    if (!isActive) {
+      // set a localstorage key to prevent banner from showing again
+      localStorage.setItem(IS_SIDE_BANNER_VISIBLE_KEY, 'false')
+    }
+  }, [isActive])
```

Comment:
In this case (saving to local storage), do you still need the local state?
I think only setting the local storage inside the `handleClose` fn should be enough, no?

Reply:
Won't work as I though, ignore :)